modules/web/js/ballerina/diagram/views/default/components/image-util.jsx

```diff
@@ -63,6 +65,39 @@ class ImageUtils {
     static getSVGIconString(iconName) {
         return images[iconName];
     }
+
+    /**
+     * Get the connector icon from service
+     * @param {string} connectorName - connector name
+     * @returns {object} - server response data
+     */
+    static getConnectorIcon(connectorName) {
+        const requestJson = {
+            connectorName: connectorName,
+            iconPath: 'testPath',
+        };
+        let data = {};
+        $.ajax({
+            type: 'POST',
+            context: this,
+            url: 'http://localhost:8289/file/connector/icon',
```

Comment:
can we use '/file/connector/icon' here so any future port changes will not brake this?